django/utils/dateparse.py

```diff
@@ -137,11 +137,11 @@ def parse_duration(value):
     )
     if match:
         kw = match.groupdict()
-        days = datetime.timedelta(float(kw.pop('days', 0) or 0))
         sign = -1 if kw.pop('sign', '+') == '-' else 1
         if kw.get('microseconds'):
             kw['microseconds'] = kw['microseconds'].ljust(6, '0')
         if kw.get('seconds') and kw.get('microseconds') and kw['seconds'].startswith('-'):
             kw['microseconds'] = '-' + kw['microseconds']
-        kw = {k: float(v) for k, v in kw.items() if v is not None}
+        kw = {k: float(v.replace(',', '.')) for k, v in kw.items() if v is not None}
+        days = datetime.timedelta(kw.pop('days', .0) or .0)
         return days + sign * datetime.timedelta(**kw)
```

Comment:
Hey @felixxm, I have a doubt rather a clarification, I found some variables a little less informative in terms of what they are doing, like the  variable `kw` which I believe is coming from `kwargs` but wouldn't it make more sense if we name it something like `time_map` or `date_info`?

Same in dictionary comprehension that we are using.

Is there are specific reason we kept it like that?

Just curious about it :) Thanks in advance

Reply:
They aren't misleading IMO and we try to avoid trivial changes.